docs/book/src/self-managed/custom-vnet.md

```diff
@@ -141,22 +146,22 @@ spec:
           name: my-subnet-cp-nsg
           securityRules:
             - name: "allow_ssh"
-              description: "allow SSH"
+              description: "Deny SSH"
```

Comment:
Why are we updating this example to a `Deny` **SSH** ?

Reply:
I wanted to showcase that even though port 22 is defaulted, the rule for this can be overridden to deny it.

Reply:
I think it would benefit if we show an overriding scenario the example you added that adds `9345` to the APIServer LB.

Updating the rule to Deny could throw off users in the first glance.